packages/atlas/codegen.config.ts

```diff
@@ -1,8 +1,13 @@
 import { CodegenConfig } from '@graphql-codegen/cli'
+import * as assert from 'assert'
 
 import { customSchemaLoader } from './scripts/customSchemaLoader'
 
-const schemaUrl = 'https://orion.gleev.xyz/graphql'
+const NODE_ENV = process.env.NODE_ENV?.toUpperCase() ?? 'PRODUCTION'
```

Comment:
I think this should be either `VITE_ENV` or `VITE_DEFAULT_DATA_ENV`

Reply:
Yes `VITE_DEFAULT_DATA_ENV` would make a lot of sense